internal/source/unpacker.go

```diff
@@ -54,6 +56,9 @@ type Result struct {
 	// Message is contextual information about the progress of unpacking the
 	// catalog content.
 	Message string
+
+	// LastTraansitionTime is the timestamp when the transition to the current State happened
+	LastTransitionTime metav1.Time
```

Comment:
I think this should be a regular timestamp. We can convert to `metav1.Time` when we're in the kubernetes object domain. This library isn't in that domain.
```suggestion
	UnpackTime time.Time
```